lib/pgslice.rb

```diff
@@ -518,8 +521,13 @@ def retired_name(table)
       "#{table}_retired"
     end
 
-    def sql_date(time)
-      "'#{time.strftime("%Y-%m-%d")}'::date"
+    def column_cast(table, column)
+      data_type = execute("SELECT data_type FROM information_schema.columns WHERE table_name = $1 AND column_name = $2", [table, column])[0]["data_type"]
```

Comment:
Let's add a `AND table_schema = 'public'` here since PgSlice uses `public` in other places (can make this configurable in the future)